app/components/images/imagesController.js

```diff
@@ -59,7 +59,28 @@ function ($scope, $state, Config, Image, ImageHelper, Messages, Pagination) {
     });
   };
 
-  $scope.removeAction = function () {
+  $scope.confirmRemovalAction = function (force) {
+    bootbox.confirm({
```

Comment:
Do you think you could isolate the bootbox call inside a specific service ? Like `ModalService` ? This way if we want to get rid of `bootbox` later, we'll just have to change the implementation there.